frontend/src/components/UI/NavBar.tsx

```
@@ -29,76 +31,109 @@ import { UserTypeEnum } from "../../types/user";
+    // Opcional: actualizar el hash en la URL sin scroll
+    if (!menuOpen) {
+      history.replaceState(null, "", window.location.pathname + "#menu");
+    } else {
+      history.replaceState(null, "", window.location.pathname);
+    }
+  };
+    const user_type: UserTypeEnum = User.data.type;
     const buttons: React.ReactElement[] | undefined[] = [
-        <img src={notification_icon} alt="Notification" className="bell" style={{ display: window.innerWidth > window.innerHeight ? "none" : "block"}} />,
+        <img src={notification_icon} alt="Notification" className="bell" style={{ display: window.innerWidth > window.innerHeight ? "none" : "block" }} />,
         <a href="#footer"><img src={about_us_icon} alt="About Us" /></a>,
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)